### ISSUE TITLE
Alert when the set of people who can access cloud.gov IaaS resources is in an unexpected state

### ISSUE THREAD

**Issue by mogul** [18F/cg-atlas#10]
In order to minimize the risk of cloud.gov&nbsp;infrastructure&nbsp;being tampered with at the AWS level, only members of the cloud.gov team should be able to access cloud.gov AWS resources.

**Acceptance Criteria**
- [x] We have a test for only appropriate people in the cloud.gov team being able to access cloud.gov resources in AWS
- [ ] The test shows no problems when we run it
- [ ] The test is running on a regular basis and sends alerts if it fails

(This refers to the fact that other people at 18F outside the team retain some access as we've never detangled cloud.gov from non-cloud.gov. That access needs to be restricted, whether via negative IAM, or totally separate AWS accounts.)

**Comment by cnelson** [comment]
Since cloud.gov has it's own account in GovCloud, do we need to do anything else to close this out?

**Comment by mogul** [comment]
Nope, I think that demoing #25 (redeploy in GovCloud) basically knocks this one straight to "completed", because we're doing this:

> That access needs to be restricted, whether via negative IAM, or **totally separate AWS accounts**.

**Comment by mogul** [comment]
BUT, please don't close this issue until that's done... Our Readiness Assessment Report refers specifically to this issue as where to track the state of the remediation.

**Comment by mogul** [comment]
I set the estimate on this one to "none" since it's zero effort based on #25 completing and should not be factoring into our burndown.

**Comment by mogul** [comment]
I updated the ACs on this one to reflect our intention to not only resolve the existing situation, but have a test to prevent it from recurring in future. I also raised the estimate from a 0 to a 1 to reflect that it's no longer a no-op resolved via the redeployment to GovCloud.

**Comment by mogul** [comment]
Test exists, but no alerts yet. Steve wants to add some additional conditions to test for.

**Comment by mogul** [comment]
(Feel free to finish this @sharms, but I'm disconnecting it from the epic about FedRAMP remediations since we're in a clean state right now.)
